go/oasis-node/cmd/debug/byzantine/tendermint.go

```diff
@@ -37,7 +37,7 @@ func (ht *honestTendermint) start(id *identity.Identity, dataDir string) error {
 	}
 	genesisDoc.SetChainContext()
 
-	ht.service, err = tendermint.New(context.Background(), dataDir, id, genesis)
+	ht.service, err = tendermint.New(context.Background(), dataDir, id, nil, genesis) // XXX
```

Comment:
What's with the XXX? I guess it should be fine if the byzantine node doesn't support upgrades ;-)